.changeset/warm-houses-switch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@biomejs/biome": patch
+---
+
+Fixed [#6783](https://github.com/biomejs/biome/issues/6783): stdin formatting using `--stdin-file-path` is no longer blocked by `files.includes` when the provided path doesn't exist on disk.
```

Comment:
```suggestion
Fixed [#6783](https://github.com/biomejs/biome/issues/6783): now, when a path is provided via `--stdin-file-path`, Biome checks whether the file exists within the current project. If the path doesn't exist, ignore checks - `files.includes` and VCS-ignore files - are skipped, and the file is handled. 
```

What do you think of this suggestion? I think it is a bit clearer for our end users. Feel free to change it, but the idea is that we need to mention VCS too, and making the phrase generic towards stdin, not formatting.

Reply:
@tt-a1i what do you think of this suggestion?